frontend/src/views/Lobby.tsx

```diff
@@ -331,6 +333,25 @@ function LobbyPage() {
       });
   };
 
+  const updateSize = () => {
+    setLoading(true);
+    const prevSize = size;
+    const settings = {
+      initiator: currentUser!,
+      size: size,
+    };
+
+
+    updateRoomSettings(currentRoomId, settings)
+      .then(() => setLoading(false))
```

Comment:
Also add `setError('')` here, as sometimes, the room size can show an error even when not necessary ([screencast showing this](https://www.loom.com/share/8ef1a6d5d499480ca59aa7ed3e4145bf)).